gems/manageiq_ansible_tower/manageiq_ansible_tower.gemspec

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'manageiq_ansible_tower/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = "manageiq_ansible_tower"
+  spec.version       = ManageiqAnsibleTower::VERSION
+  spec.authors       = ["Brandon Dunne"]
+  spec.email         = ["bdunne@redhat.com"]
+
+  spec.summary       = %q{Ansible Tower Provider for ManageIQ}
+  spec.description   = %q{Ansible Tower Provider for ManageIQ}
```

Comment:
Ansible itself doesn't provide a Ruby REST API gem (according to @bdunne), so this, *I think*, could become the official one.  As such, putting Manageiq in the name seems completely wrong to me.